src/handbook/customer/customer-success.md

```diff
@@ -4,9 +4,37 @@ navTitle: Customer Success
 
 # Customer Success
 
-Customer Success (CS) helps our customers get value from FlowFuse. This should
-result in them continuing to use, make broader use of, and become advocates of
-FlowFuse.
+FlowFuse CSMs are to drive adoption and expansion within accounts. Customers
```

Comment:
@Esmewieringa This is the real change, almost all others are formatting changes.